docker/download-mutinynet.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
We should document usage of this shell script in the README, otherwise it probably won't be obvious to people that this convenient thing exists.

However, I'm a little confused as to what this is for. If you have this shell script from this repo, you also have the `docker-compose.yml` files right here. Why are we downloading them? Is this meant to be copy and pasted and run outside of this repo? If so, this feels like something that should be in its own quickstart / docker repo.

Reply:
The goal is to put this in the documentation:
`curl -sSL https://raw.githubusercontent.com/fedimint/fedimint/master/docker/download-mutinynet.sh | bash`

Then people will just copy and paste this line.